pages/health-care/health-needs-conditions/mental-health.md

```diff
@@ -22,7 +22,7 @@ Find out how to access VA mental health services for posttraumatic stress disord
 <div class="usa-alert usa-alert-warning">
   <div class="usa-alert-body">
 	<h4 class="usa-alert-heading">How do I talk to someone right now?</h4>
-	<a id="crisis-expander-link">Find out how to get support anytime, day or night</a></h4>
+	<a id="crisis-expander-link" onClick="window.dataLayer.push({event: 'nav-crisis-get-support-247'});">Find out how to get support anytime, day or night</a></h4>
```

Comment:
We actually have a global helper method, `recordEvent`, that works as an alias for `window.dataLayer.push`. I think it's linked in the site header, and used throughout other content files.

Reply:
Thanks! Replaced all `window.dataLater.push` with `recordEvent`.